### PR TITLE
Events ownership

### PR DESCRIPTION
**Pull request by luizbranco**
Following @jelmersnoeck suggestions about OwnerData and use a pointer for Data.

I also added Email to Actor and Scope. It can be either a user or a team, maybe a team will have an email contact in the future, but the cli filters empty values if that isn't the case.

### CODE REVIEW

File: events/events.go [hunk 1]
```diff
@@ -343,7 +343,26 @@ func (b *BaseBody) SetIPAddress(ip string) {
...
+}
+
+// SetOwner sets the owner depending if it is a user or a team.
+func (o *OwnerData) SetOwner(id manifold.ID) {
+	switch id.Type() {
+	case idtype.User:
+		o.UserID = &id
+	case idtype.Team:
+		o.TeamID = &id
+	}
```
Comment (jelmersnoeck) [inline events/events.go]: Just wondering if we should panic here if it's neither of these options? I'd rather have it panic when we give it an ID that isn't supposed to be passed through than having data missing.

Comment (jelmersnoeck) [reply events/events.go]: (If you do, add a comment about that in the comment ;-) )

Comment (luizbranco) [reply events/events.go]: sure `panic("danger will robinson")`

File: events/events.go [hunk 2]
```diff
@@ -357,11 +357,13 @@ type OwnerData struct {
 
 // SetOwner sets the owner depending if it is a user or a team.
 func (o *OwnerData) SetOwner(id manifold.ID) {
-	switch id.Type() {
+	switch t := id.Type(); t {
```
Comment (jelmersnoeck) [inline events/events.go]: Never thought of this, that's pretty genius.

**Pull request merged**